src/app/beer_garden/api/http/handlers/v1/command_publishing_blocklist.py

```
@@ -0,0 +1,140 @@
+
+        self.write(response)
+
+    def post(self):
+        """
+        ---
+        summary: Add a list of commands to event publishing block list
+        parameters:
+          - name: CommandPublishingBlocklist
+            in: body
+            description: The system, namespace and command name
+            schema:
+              type: array
+              items:
+                  $ref: '#/definitions/CommandPublishingBlocklist'
+        consumes:
+          - application/json
+        responses:
+          201:
+            description: Command has been added to publishing block list
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklist'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Command Block List
+        """
+        errors = []
+        for command in self.request_body:
+            systems = System.objects(
+                namespace=command["namespace"], name=command["system"]
+            )
+            try:
+                self.get_or_raise(System, SYSTEM_UPDATE, id=systems[0].id)
+                if config.get("garden.name") != command["namespace"]:
+                    blocked_command = beer_garden.router.route(
+                        Operation(
+                            operation_type="COMMAND_BLOCK_LIST_ADD",
+                            kwargs={"command": command},
+                            target_garden_name=config.get("garden.name"),
+                        )
+                    )
+                    command["id"] = blocked_command._data["id"].__str__()
+
+                beer_garden.router.route(
+                    Operation(
+                        operation_type="COMMAND_BLOCK_LIST_ADD",
+                        kwargs={"command": command},
+                        target_garden_name=command["namespace"],
+                    )
+                )
+            except Exception as e:
```

Generic note:  Don't catch all exceptions.